tachyon/zk/plonk/examples/example_circuit_tests.cc

```diff
@@ -0,0 +1,477 @@
+#include "tachyon/zk/plonk/examples/example_circuit_tests.h"
+
+#include <utility>
+#include <vector>
+
+#include "gtest/gtest.h"
+
+#include "tachyon/base/array_to_vector.h"
+#include "tachyon/zk/plonk/halo2/pinned_constraint_system.h"
+#include "tachyon/zk/plonk/halo2/pinned_verifying_key.h"
+#include "tachyon/zk/plonk/keys/proving_key.h"
+
+namespace tachyon::zk::plonk::halo2 {
+
+template <typename TestingArguments, typename TestData>
```

Comment:
```suggestion
template <typename TestArguments, typename TestData>
```
It's an optional request for change, but how about like this for consistency with `TestData`?